### DISCUSSION TITLE
Connection to local XAMPP Mysql not working

### DISCUSSION THREAD

**Discussion by Solaris42** [Sequel-Ace/Sequel-Ace#1454] [General] [answered]
Hi,
after long time useing Sequel Pro, now I change to MacBook M1 Max and I want to change to Sequel Ace.
Because the support for Sequel Ace will be continue and it's easy to migrate all connections.

I was trying all workarounds, but I can't connect lokal database on XAMPP.
I read about the sandbox problem...
Is there any solution into the pipeline?

Regards 
Ralf

PS. Is there any possibility to donate? I love to work with Sequel Pro and I will love more to work with Sequel Ace :-)

**Comment by Jason-Morcos** [comment, marked as answer]
Hey, @Solaris42! I think unfortunately XAMPP is still not supported with the sandbox problem with no real solution in the pipeline. The sandbox is quite complex!

As to donation, yes there is! You can sponsor one of the lead maintainers directly or donate to the project's OpenCollective via the main Repository page! https://github.com/Sequel-Ace/Sequel-Ace
<img width="455" alt="Screen Shot 2022-04-28 at 15 31 59" src="https://user-images.githubusercontent.com/10710367/165858483-e1cfa9bc-7f0e-4154-bb49-df027cb28025.png">
